ios/Capacitor.podspec

```diff
@@ -16,8 +16,8 @@ Pod::Spec.new do |s|
   s.ios.deployment_target = '13.0'
   s.authors = { 'Ionic Team' => 'hi@ionicframework.com' }
   s.source = { git: 'https://github.com/ionic-team/capacitor.git', tag: package['version'] }
-  s.source_files = "#{prefix}Capacitor/Capacitor/*.{swift,h,m}", "#{prefix}Capacitor/Capacitor/Plugins/*.{swift,h,m}",
-                   "#{prefix}Capacitor/Capacitor/Plugins/**/*.{swift,h,m}"
+  s.source_files = "#{prefix}Capacitor/Capacitor/*.{swift,h,m}", "#{prefix}Capacitor/Capacitor/Codable/*.{swift,h,m}",
```

Comment:
I know this is late, but why don't we just make this:
```
s.source_files = "#{prefix}Capacitor/Capacitor/**.{swift,h,m}"
```
This should recursively include all files under capacitor and we wouldn't have to keep adding this anytime we decide to add a folder for organization purposes.